internal/pfcp/session.go

```diff
@@ -84,15 +84,33 @@ func (s *PfcpServer) handleSessionEstablishmentRequest(
 			sess.log.Errorf("Est CreatePDR error: %+v", err)
 		}
 
-		ueIPAddress := getUEAddressFromPDR(i)
 		pdrId := getPDRIDFromPDR(i)
-
-		if ueIPAddress != nil {
-			ueIPv4 := ueIPAddress.IPv4Address.String()
-			CreatedPDRList = append(CreatedPDRList, ie.NewCreatedPDR(
-				ie.NewPDRID(pdrId),
-				ie.NewUEIPAddress(2, ueIPv4, "", 0, 0),
-			))
+		if pdrId != 0 {
```

Comment:
@ianchen0119 @williamlin0518 CreatedPDR in Est rsp is not mandatory and upf will not allocate F-TEID or UE IP, so no need to build CreatedPDR in Est rsp. So, please remove line 87~113